lib/Dialect/TritonNvidiaGPU/Transforms/MMALowering.cpp

```diff
@@ -37,7 +37,8 @@ class SyncMMALowering : public OpInterfaceRewritePattern<MMAv5OpInterface> {
                               sharedMemorySpace, /*mutableMemory=*/true);
     Value barrierAlloc =
         ttg::LocalAllocOp::create(rewriter, loc, barrierMemDescType, Value());
-    InitBarrierOp::create(rewriter, loc, barrierAlloc, 1);
+    InitBarrierOp::create(rewriter, loc, barrierAlloc, 1,
+                          DenseI32ArrayAttr({}));
```

Comment:
please add a builder so that you don't need to update every single create callsite